lib/vintage_net_wizard/backend.ex

```diff
@@ -150,6 +160,13 @@ defmodule VintageNetWizard.Backend do
     GenServer.call(__MODULE__, :apply)
   end
 
+  @doc """
+  Load the configs that might already exist on the system
+  """
+  def load_configurations() do
```

Comment:
This needs a spec. It looks like this returns a map instead of a list, though, like the behaviour callback function of the same name.